### PR TITLE
Update all docs demos to have autowrap

### PR DESCRIPTION
**Pull request by evanSe**
### Description
Update documentation to present the best accessibility experience out of the box, removal of progress, stars and row header checkbox and set autowrapping to true on all demos.

```
autoWrapRow: true,
autoWrapCol: true,
```

- [x] remove all progress cell types from demos
- [x] Remove all star cell types from the demo
- [x] Remove all row header checkboxes from demo's 
- [x] `autoWrapRow: true` on all demos
- [x] `autoWrapCol: true` on all demos

[skip changelog]

### CODE REVIEW

File: docs/content/guides/cell-functions/cell-renderer.md [hunk 1]
```diff
@@ -99,7 +99,10 @@ const hotData = [
 
 export const ExampleComponent = () => {
   return (
-    <HotTable data={hotData} licenseKey="non-commercial-and-evaluation">
+    <HotTable data={hotData}
+    autoWrapRow={true}
+    autoWrapCol={true}
+    licenseKey="non-commercial-and-evaluation">
```
Comment (budnix) [inline docs/content/guides/cell-functions/cell-renderer.md:105]:
```suggestion
    <HotTable data={hotData}
      autoWrapRow={true}
      autoWrapCol={true}
      licenseKey="non-commercial-and-evaluation">
```

File: docs/content/guides/columns/column-header.md [hunk 2]
```diff
@@ -168,7 +176,7 @@ import Handsontable from 'handsontable';
 import 'handsontable/dist/handsontable.full.min.css';
 
 const container = document.querySelector('#example3');
-const hot = new Handsontable(container, {
+const hot = sontable(container, {
```
Comment (budnix) [inline docs/content/guides/columns/column-header.md:179]:
```suggestion
const hot = new Handsontable(container, {
```
😅

File: docs/content/guides/columns/column-moving.md [hunk 3]
```diff
@@ -86,6 +88,8 @@ export const ExampleComponent = () => {
       colHeaders={true}
       colWidths={100}
       manualColumnMove={true}
+      autoWrapRow={true}
+      autoWrapCol={true }
```
Comment (budnix) [inline docs/content/guides/columns/column-moving.md:92]:
```suggestion
      autoWrapCol={true}
```

File: docs/content/guides/columns/column-virtualization.md [hunk 4]
```diff
@@ -99,7 +101,9 @@ const data = new Array(1000) // number of rows
   );
 
 export const ExampleComponent = () => {
-  return <HotTable data={data} colWidths={100} width="100%" height={320} rowHeaders={true} colHeaders={true} licenseKey="non-commercial-and-evaluation" />;
+  return <HotTable data={data} colWidths={100} width="100%" height={320} rowHeaders={true} colHeaders={true} autoWrapRow={true}
+  autoWrapCol={true}
+  licenseKey="non-commercial-and-evaluation" />;
```
Comment (budnix) [inline docs/content/guides/columns/column-virtualization.md:106]:
```suggestion
  return <HotTable data={data} colWidths={100} width="100%" height={320} rowHeaders={true} colHeaders={true} autoWrapRow={true}
    autoWrapCol={true}
    licenseKey="non-commercial-and-evaluation" />;
```

File: docs/content/guides/columns/react-hot-column.md [hunk 5]
```diff
@@ -49,7 +49,10 @@ const secondColumnSettings = {
 
 export const ExampleComponent = () => {
   return (
-    <HotTable data={hotData} licenseKey="non-commercial-and-evaluation">
+    <HotTable data={hotData} 
+    autoWrapRow={true}
+    autoWrapCol={true}
+    licenseKey="non-commercial-and-evaluation">
```
Comment (budnix) [inline docs/content/guides/columns/react-hot-column.md:55]:
```suggestion
    <HotTable data={hotData} 
      autoWrapRow={true}
      autoWrapCol={true}
      licenseKey="non-commercial-and-evaluation">
```

File: docs/content/guides/getting-started/binding-to-data.md [hunk 6]
```diff
@@ -757,7 +787,10 @@ registerAllModules();
 
 export const ExampleComponent = () => {
   return (
-    <HotTable licenseKey="non-commercial-and-evaluation" />
+    <HotTable 
+    autoWrapRow={true}
+    autoWrapCol={true}
+    licenseKey="non-commercial-and-evaluation" />
```
Comment (budnix) [inline docs/content/guides/getting-started/binding-to-data.md:793]:
```suggestion
    <HotTable 
      autoWrapRow={true}
      autoWrapCol={true}
      licenseKey="non-commercial-and-evaluation" />
```

**Review by budnix — CHANGES_REQUESTED** [review]

**Review by budnix — CHANGES_REQUESTED** [review]
I'd say that it looks good but the CI says something different https://github.com/handsontable/handsontable/actions/runs/7276585493/job/19826734582?pr=10677.

**Comment by evanSe** [comment]
> I'd say that it looks good but the CI says something different https://github.com/handsontable/handsontable/actions/runs/7276585493/job/19826734582?pr=10677.

Cheers, probably syntax error due to how much loc changed

**Review by budnix — APPROVED** [review]

**Pull request merged**